ultralytics/cfg/__init__.py

```diff
@@ -546,6 +547,71 @@ def handle_yolo_settings(args: List[str]) -> None:
         LOGGER.warning(f"WARNING ⚠️ settings error: '{e}'. Please see {url} for help.")
 
 
+def handle_yolo_info(args: List[str]) -> None:
+    """Handles YOLO info command-line interface (CLI) commands."""
+    from ultralytics import NAS, RTDETR, SAM, YOLO, FastSAM
+
+    delim = "-" * 80
```

Comment:
```suggestion
    delim = "-" * os.get_terminal_size().columns - 4
```

Reply:
I don't know if this also works for other system than linux.

Reply:
No worries, it works!

![image](https://github.com/user-attachments/assets/628b8a79-7af6-458d-920d-79239f4d8ed1)

I wrote this and tested it on Windows and the CI is passing for all OSes.